modules/ROOT/pages/bonita-bpm-overview.adoc

```diff
@@ -1,81 +1,94 @@
+Bonita Studio is the development tool for both the Citizen developers and the Professional Developers. +
+Citizen developers (Business Analysts, Business Project Managers, Power Users) work in the Studio and the UI Designer to input business-level information to the project: application skeletons, processes, data model, and high-level user-interfaces. +
+As for the Professional developers, the back-end Java developers work in both the Studio and their favorite IDE, and the front-end developers work in the Studio, the UI Designer as well as their favorite IDE. +
 
-But Bonita Portal is also the tool used by the tenant administrator to xref:processes.adoc[install, deploy and manage processes] and to xref:applications.adoc[build applications]. We use the word "tenant" here, in case the platform would be made of xref:multi-tenancy-and-tenant-configuration.adoc[several tenants].
 
-To install Bonita Engine and Bonita Portal in a qualification or production environment, xref:bonita-bpm-installation-overview.adoc#platform[install Bonita Platform].
+[.card-section]
+=== Related documentation pages
 
-In Bonita Studio, once a process is ready, you can xref:build-a-process-for-deployment.adoc[build] it and deploy it on your Bonita qualification or production platform.
 
-== Editions
+[.card.card-index]
+--
+xref:bonita-studio.adoc[[.card-title]#Bonita Studio# [.card-body.card-content-overflow]#pass:q[Create process-based applications for business users]#]
+--
 
-Until 2019 Bonita was provided in four different editions: Community, Teamwork, Efficiency and Performance. Since 2019 Bonita is only provided in two editions : Community and Enterprise.
+[.card.card-index]
+--
+xref:ui-designer-overview.adoc[[.card-title]#Bonita UI Designer# [.card-body.card-content-overflow]#pass:q[Create the User Interfaces of you applications]#]
+--
 
-== Getting started
+[.card.card-index]
+--
+xref:bonita-studio-download-installation.adoc[[.card-title]#Studio installation# [.card-body.card-content-overflow]#pass:q[First step to use Bonita Studio]#]
+--
 
-*I'm currently using Bonita 6.x. Will 7.x be a big change?*
+Bonita Studio also embeds a *Bonita Runtime* (Tomcat, UI Designer, Bonita provided applications, Bonita Engine, and an h2 database), suitable for testing an application that is in development. When you run a process, it is automatically deployed onto the development platform.
 
-Bonita 7.x provides new and improved features, including the xref:ui-designer-overview.adoc[UI Designer] for creating application pages and forms, as well as xref:contracts-and-contexts.adoc[contracts and context] to create a clear split between the process logic, the data, and the user views.
+Then, once a process is ready, you can xref:build-a-process-for-deployment.adoc[build] it and deploy it on your Bonita qualification or production Runtime environment. +
```

Comment:
I'm not sure it's relevant to talk about building and deploying a 'process'. Today we talk about building and deploying applications, which include related processes, business data model, profiles, organization, and any required artifact. 
I d'ont know if we have a page that explain how to manually deploy a whole Bonita application (since BCD is for subscription users), but I definitively think that we should not encourage users to only deploy process, especially on an high level page.